build/webpack/webpack.config.js

```diff
@@ -0,0 +1,170 @@
+/*
+This is just porting all the configs from all the seperate transform files and
+having one large standard one. This should make it easier to update to webpack 2
+As well as just making it more accessible without having to dive around all the
+transforms. Would prefer to have a well commented file like Origami:
+*/
+
+const path =require('path');
+const glob = require('glob');
+const BowerWebpackPlugin = require('bower-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const autoprefixer = require('autoprefixer');
+
+
+const handlebarsConfig = () => {
+	const extraHelperDirs = glob.sync('**/node_modules/@financial-times/**/handlebars-helpers')
+		.map(dir => path.resolve(dir));
+	return {
+		debug: false, // set to true to debug finding partial/helper issues
+		extensions: ['.html'],
+		helperDirs: [
+			path.resolve('./node_modules/@financial-times/n-handlebars/src/helpers'),
+			path.resolve('./server/helpers'),
+			path.resolve('./bower_components/n-concept/handlebars-helpers')
+		].concat(extraHelperDirs),
+		partialDirs: [
+			path.resolve('./bower_components'),
+			path.resolve('./node_modules/@financial-times'),
+			path.resolve('./views/universal')
+		]
+	};
+};
+
+module.exports = () => ({
+	devtool: 'source-map',
+
+	resolve: {
+		root: [
+			path.resolve('./bower_components'),
+			path.resolve('./node_modules')
+		],
+		alias: Object.assign(require('babel-polyfill-silencer/aliases'), {
+			'react': 'preact-compat',
+			'react-dom': 'preact-compat'
+		}),
+	},
+
+	module: {
+		loaders: [
+			//babel
+			{
+				test: /\.js$/,
+				loader: require.resolve('babel-loader'),
+				include: [
+					/bower_components/,
+					path.resolve('./node_modules/@financial-times/n-handlebars/src/helpers'),
+					path.resolve('./server/helpers'), // more handlebars helpers
+					path.resolve('./client'),
+					path.resolve('./config'),
+					path.resolve('./shared'),
+					/@financial-times\/n-card/,
+					/@financial-times\/n-email-article/,
+					/@financial-times\/n-image/,
+					/@financial-times\/n-myft-ui/,
+					/@financial-times\/n-notification/,
+					/@financial-times\/n-section/,
+					/@financial-times\/n-ui/,
+					/@financial-times\/n-teaser/,
+					/@financial-times\/n-counter-ad-blocking/,
+					/@financial-times\/n-native-ads/,
+					/@financial-times\/n-tourtip/,
+					/.*/
+				],
+				exclude: [
+					/node_modules/
+				],
+				query: {
+					babelrc: false, // ignore any .babelrc in project & dependencies
+					cacheDirectory: true,
+					plugins: [
+						require.resolve('babel-plugin-add-module-exports', true),
+						[
+							require.resolve('babel-plugin-transform-runtime'),
+							{ polyfill: false }
```

Comment:
If you change this to:
```
{
	helpers: false,
	polyfill: false,
	regenerator: true,
}
```
Then you should not get any core-js polyfills in your code 🎉